src/services/chris_integration.tsx

```diff
@@ -303,35 +316,32 @@ class ChrisIntegration {
             // TODO: investigate else case
             return [[], curOffset, isAtEndOfFeeds];
           }
-  
+
           const pluginInstanceFiles = await plugin.getFiles({
             limit: 25,
             offset: 0,
           });
-  
           const newSeries: ISeries = {
             covidnetPluginId: plugin.data.id,
-            imageName: '',
-            imageId: '',
+            imageName: plugin.data?.title ? plugin.data?.title : "File name not available",
```

Comment:
This might be a bit shorter and cleaner.

```suggestion
            imageName: plugin?.data?.title || "File name not available",
```